src/com/seafile/seadroid2/cameraupload/StubContentProvider.java

```diff
@@ -0,0 +1,66 @@
+package com.seafile.seadroid2.cameraupload;
+
+import android.content.ContentProvider;
+import android.content.ContentValues;
+import android.database.Cursor;
+import android.net.Uri;
+
+/**
+ * This stub content provider is just glue code between the Authenticator
+ * and the SyncProvider
+ */
+public class StubContentProvider extends ContentProvider {
```

Comment:
>The sync adapter framework expects that an app that uses the framework has already defined a content provider for its local data.

I think we already defined a content provider `SeafileProvider`, and the camera sync module indeed fetches files from it, so why bother to use this stub provider?